src/main/java/org/opentripplanner/standalone/config/updaters/sources/VehicleRentalSourceFactory.java

```diff
@@ -83,10 +82,6 @@ private String network() {
     return c.asText("network", null);
   }
 
-  private boolean routeAsCar() {
```

Comment:
Is this going to be derived from the form factor instead?

Reply:
Yes, [here](https://github.com/opentripplanner/OpenTripPlanner/pull/3610/files#diff-ef104df0cef940ccc17f4c38792e4ca004e5d9a0d150d9f4b19fb07bd81afccfR111)